test/cljam/t_common.clj

```diff
@@ -1,11 +1,35 @@
 (ns cljam.t-common
-  (:use [clojure.java.io :only [file]])
   (:require [digest]
+            [clojure.java.io :refer [file]]
             [cljam.sam :as sam]
             [cljam.bam :as bam]
             [cljam.io :as io]
             [cavia.core :as cavia :refer [defprofile with-profile]]))
 
+(defn- _in-cloverage? []
+  (let [in? (try
+              (eval '(var cloverage.coverage/*covered*))
+              true
+              (catch Throwable _ nil))]
+    ;; (prn :debug 'in-cloverage? in?)
```

Comment:
Is this line a debug print? If so, please remove the line and make this function simple.